rllib/core/learner/learner.py

```diff
@@ -352,7 +352,7 @@ def _configure_optimizers_per_module_helper(
             pairs.append(pair)
         elif isinstance(pair_or_pairs, dict):
             # pair_or_pairs is a NamedParamOptimizerPairs
-            for name, pair in pairs.items():
+            for name, pair in pair_or_pairs.items():
```

Comment:
This was a bug, but not visible for Learners that only use the default (single) optimizer path.